package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@interlay/polkabtc",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
@nud3l suggested we remove this version bump from the commits. In my opinion we should have this updated on the dev chain too, not only in the tag

Reply:
Yeah, let's update this in this PR since it will also update the parachain. But please bump to 0.2.3 since the latest release is 0.2.2